src/shared/utilities/promptSetting.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { getLogger } from '../logger'
+import { SettingsConfiguration } from '../settingsConfiguration'
+
+export type PROMPT_NAME =
+    | 'warnBeforeRunCommand'
+    | 'warnBeforeEnablingExcecuteCommand'
+    | 'warnBeforeDisablingExecuteCommand'
+export const DEFAULT_PROMPTS: PromptSetting = {
+    warnBeforeRunCommand: true,
+    warnBeforeEnablingExcecuteCommand: true,
+    warnBeforeDisablingExecuteCommand: true,
+}
+
+export interface PromptSetting {
+    [promptName: string]: boolean
+}
+
+export function disablePrompt(promptName: PROMPT_NAME, settings: SettingsConfiguration): void {
```

Comment:
This function could be generalized to "disable" or "enable" at very little cost, so generally it's worth doing so.

Reply:
This function is now a method of DefaultSettingsConfiguration and is called `disable`